frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -54,7 +60,7 @@ describe('Kubevirt create VM using wizard', () => {
   });
 
   afterAll(async () => {
-    deleteResources([multusNAD, testDataVolume]);
```

Comment:
it needs to be cleaned up, even if the tests fail. https://github.com/openshift/console/pull/6972#discussion_r514393315